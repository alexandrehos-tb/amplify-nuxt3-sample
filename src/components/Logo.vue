<template>
  <NuxtLink :to="Url.TOP" >
    <v-toolbar-title class="w-64">nuxt3-typescript-aws</v-toolbar-title>
  </NuxtLink>
</template>

<script setup lang="ts">
// import { useI18n } from 'vue-i18n'
// const { t } = useI18n()
import { Url } from '@/constants/url'
</script>