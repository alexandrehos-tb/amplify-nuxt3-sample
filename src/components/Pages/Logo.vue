<template>
  <NuxtLink :to="Url.TOP">
    <v-toolbar-title class="w-64"> amplify-nuxt3-sample </v-toolbar-title>
  </NuxtLink>
</template>

<script setup lang="ts">
import { Url } from '@/constants/url'
</script>
