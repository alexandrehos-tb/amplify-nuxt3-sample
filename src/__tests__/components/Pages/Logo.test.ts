import { mount } from '@vue/test-utils'
import Logo from '@/components/Pages/Logo.vue'

test('Logo', () => {
  const wrapper = mount(Logo, {
    global: {
    },
  })
  expect(wrapper.element).toMatchSnapshot()
})