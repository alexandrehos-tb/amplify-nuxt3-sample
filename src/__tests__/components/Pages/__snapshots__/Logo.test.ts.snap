// Vitest Snapshot v1

exports[`Logo 1`] = `
<nuxtlink
  to="/"
>
  <v-toolbar-title
    class="w-64"
  >
     amplify-nuxt3-sample 
  </v-toolbar-title>
</nuxtlink>
`;
